lib/ReduxInfiniteScroll.js

```diff
@@ -67,7 +69,7 @@ var ReduxInfiniteScroll = function (_React$Component) {
   }, {
     key: '_elScrollListener',
     value: function _elScrollListener() {
-      var el = _reactDom2.default.findDOMNode(this);
+      var el = this.props.scrollableElement || document.getElementById('workspace');
```

Comment:
do you think we can get rid of this workspace thing and open a PR for the library ? Would be nice to drop some open source contributions 😄

Reply:
Good catch - this is not meant to be there at all! 
Yeah I'll prepare the PR, however the author hasn't responded in 6 months and this is based on 2 pending PRs for the library already :\